### PR TITLE
Change Travis from Julia 1.4 to 1.5

### PR DESCRIPTION
**Pull request by bkamins**
Update after the release of Julia 1.5.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ language: julia
 
 julia:
   - 1.0
-  - 1.4
+  - 1.5
```
Comment (quinnj) [inline .travis.yml:5]:
```suggestion
  - 1
```
this shorthand will auto update when the next minor release is changed

Comment (bkamins) [reply .travis.yml:5]: A good point - thank you!

**Comment by bkamins** [comment]
@quinnj - can you please approve this, I will merge it, backport and make a release today. (I have just bumped the patch number in version this PR)

**Comment by bkamins** [comment]
I will make a release based on this commit to fix the problem with CSV.jl.

**Pull request merged**